aws/cloudformation/scylla.yaml

```diff
@@ -56,6 +56,23 @@ Parameters:
     Type: 'AWS::EC2::KeyPair::KeyName'
     ConstraintDescription: must be the name of an existing EC2 KeyPair.
 
+  CIDR:
+    Description: |
+        CIDR range will be used by Scylla VPC (ex. 172.31.0.0/16),
+        the node IPs will be x.x.x.10, x.x.x.11, x.x.x.12 etc.
```

Comment:
You assume here that the range that will be provied by the user has this ip addresses (the ending). Why not changing it so we will always use first ip in range for Node0 and so on?

Reply:
Well, you right, - the IPs (10,11,12,13 etc.) are hard coded due to CFN limitations
\-  even if I'll have the last octet (ex. `!Select [3, !Split ['.', !Select [0, !Split ['/', !Ref CIDR]]]]`), CFN has no math-operators we can use to increase the result by x,

The math-operators and other custom function can be done by `custom-resource` or `macro` which require pre-configured lambda function which is different story.